test/onnx/gen_onnx.py

```diff
@@ -5793,6 +5793,24 @@ def reshape_non_standard_test():
     return ([trans, res], [x], [y])
 
 
+@onnx_test()
+def reshape_variable_input_test0():
```

Comment:
Why is it named `variable` I don't see any `None` in dimensions.

Reply:
Means the second input is variable rather than constant (as in the other tests).